Src/JxrDecode/CMakeLists.txt

```diff
@@ -2,14 +2,64 @@
 #
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
-# This is a workaround for a problem in VisualStudio 2017 15.7. Without it, the CMake-generation in the IDE fails with a
-# spurious exception -> https://github.com/Microsoft/VSLinux/issues/288
-add_definitions(-DFOO)
```

Comment:
Yes, and this workaround is not in place in other CMakeLists.txt anymore, so I'd think we don't break anything (which isn't elsewhere already "broken") by removing it here also. I didn't test with VS2017 since quite a while.